docker/local-metadata/docker-compose.yml

```diff
@@ -122,11 +122,15 @@ services:
       # Airflow SSL Configurations
       AIRFLOW_VERIFY_SSL: ${AIRFLOW_VERIFY_SSL:-"no-ssl"}
       AIRFLOW_SSL_CERT_PATH: ${AIRFLOW_SSL_CERT_PATH:-""}
+      # Admin Port 
+      SERVER_ADMIN_PORT: ${SERVER_ADMIN_PORT:-8586}
+      # Server Port
+      SERVER_PORT: ${SERVER_PORT:-8585}
     expose:
       - 8585
       - 8586
     ports:
-      - "8585:8585"
+      - "80:8585"
```

Comment:
Please revert this change! This will break all the existing CI.

Reply:
Done